cypress/e2e/8SummarisePDF.cy.js

```diff
@@ -0,0 +1,23 @@
+describe("Cypress Testing PDF", () => {
+  it("Summarise PDF", () => {
+    // visit our beautiful website
+    cy.visit("/");
+
+    // check if the summarise button is disabled
+    cy.get("#summarise-button-id").should("be.disabled");
+
+    // wait
+    cy.wait(1000);
+
+    // call my readPdf custom function (cypress.config.ts) on task
+    cy.task(
+      "readPdf",
+      Cypress.config("fileServerFolder") + "/cypress/case-study.pdf"
+    ).then((pdfText) => {
+      // Input/Type it on to the text area
+      cy.get("#inputted-text").type(pdfText);
+
+      cy.get("#summarise-button-id").click();
```

Comment:
This test doesn't actually test the PDF input functionality, it uses the custom method defined in `readPdf.js` to parse the PDf, and then just types the extracted text into the input. 

It will need to simulate dragging and dropping a PDF file into the textarea, and then asserting that the input text has been updated, word count reflects correctly, and the summarise button is enabled. From memory there is a library called `cypress drag drop` that might be helpful here (drag and drop is a pretty common UI interaction so there should be support to test it out).

Reply:
The workflow should be:

- visit /
- drag and drop `case-studfy.pdf`
- assert input text is updated (check the textarea value and compare it to string)
- assert the word count is correct
- assert summarise button enabled

also would be good to test other file types
- drag and drop a random `.docx` file
- assert an alert appears for incorrect file type